src/main.rs

```diff
@@ -42,12 +28,14 @@ use std::path::Path;
 use protobuf::core::Message;
 use std::thread;
 
-use metadata::{AlbumRef, ArtistRef, TrackRef};
-use session::{Config, Session};
-use util::SpotifyId;
-use util::version::version_string;
-use player::{Player, PlayerCommand};
-use mercury::{MercuryRequest, MercuryMethod};
+use librespot::util;
+use librespot::metadata::{AlbumRef, ArtistRef, TrackRef};
+use librespot::session::{Config, Session};
+use librespot::util::spotify_id::SpotifyId;
```

Comment:
For some reason using `librespot::util::SpotifyId;` didn't work here, even though `util/mod.rs` has `pub use util::spotify_id::{SpotifyId, FileId};` Ideas why?